### ISSUE TITLE
Sending message back

### ISSUE THREAD

**Issue by sachanacar** [falcondai/wechat-nodejs-guide#2]
Awesome guide! Thanks a lot!

I was wondering if you knew how we could send messages back to the account that is linked to the webhook. Basically, does WeChat support incoming webhooks?

**Comment by falcondai** [comment]
i am glad that you find it helpful 😄 

do you mean a push mechanism, i.e. messaging users not as a reply to their messages?

**Comment by sachanacar** [comment]
Hey! I actually looked through the wechat-npm and found it :)

This does the trick:

```
res.reply('hello!')
```

**Comment by falcondai** [comment]
Yes, I should have covered that (but i have since moved onto other things). So WeChat uses standard HTTP request-reply to handle a user-initiated exchange. just out of curiosity, what are you working on?

**Comment by sachanacar** [comment]
I'm working on a chatbot built in Java that can strap itself onto different platforms (Slack, Spark, and Jabber, so far).

Here is the link to the project https://github.com/vmorsiani/jabbot
Here is the link to the scripting language: https://github.com/tsearle/script

So I'm trying to link it to wechat!

**Comment by falcondai** [comment]
interesting. wechat is definitely a hot market but i found its public account feature not very productivity-friendly unlike Slack.
